dist/index.html

```diff
@@ -37,6 +38,68 @@ <h1>
             </div>
         </div>
     </header>
+    <footer>
+        <div class="footer-section" id="about-us">
+            <h3>About Us</h3>
+            <p> TOP movies is a dynamic film exhibition site with a style that incorporates multifarious features, including state-of-the-art cinema technologies and luxurious dine-in cinema services; administered by a professional operations management.<br>
+                <br>We are driven by the need to continue discovering new and innovative ways of creating inspiring experiences, delivering world-class service, and bringing the magic of cinema to life. Our philosophy is defined by quality, offering luxury and the most premium service affordably, primarily to ensure you receive absolute value. We focus on offering the best service, leaving you that magical experience, every time.
```

Comment:
Instead of using <br>, you can make it a fresh paragraph for screen readers and maintaining semantic HTML